CHANGELOG.md

```diff
@@ -37,6 +37,8 @@
 - Added mesh *Triangulation* methods and *Triangulate Mesh* node.
 - Added Mesh Points Scatter node.
 - Added support for 2D vectors in 3D viewer node.
+- Added matrices output in *Mesh Points Scatter* node.
+- Added edges mode in *Mesh Points Scatter* node.
```

Comment:
Change log entries now goes to the Unreleased section above.

Reply:
Done.